src/homepageExperience/components/steps/WriteData.tsx

```diff
@@ -114,10 +120,10 @@ export const WriteDataComponent = () => {
   )
 }
 
-export const WriteData = () => {
+export const WriteData = props => {
   return (
     <WriteDataDetailsContextProvider>
-      <WriteDataComponent />
+      <WriteDataComponent {...props} />
```

Comment:
we don't need to use spread props here since this is a custom component. The typical use case is when a react component wraps an html component that takes props, like a `<InfluxButton>` wrapping a `<button>`. It's fine to pass a single prop down like this, it's more explicit and self-documenting